.github/workflows/tests.yml

```diff
@@ -26,6 +26,8 @@ jobs:
         - ["3.9",   "py39"]
         - ["3.10",  "py310"]
         - ["3.11",  "py311"]
+        - ["3.12",  "py312"]
+        - ["3.13.0-alpha - 3.13.99",  "py313"]
```

Comment:
Given the issues with Python 3.11 as demonstrated in #387, I guess supporting the unreleased Python 3.13 is a bit out of scope for this PR.

Reply:
Adding nightly to the test matrix is meant to alert early, not saying we support it.